src/shogun2-webapp-archetype/src/main/resources/archetype-resources/src/main/java/util/WmsResponseInterceptor.java

```diff
@@ -0,0 +1,48 @@
+#set( $symbol_pound = '#' )
+#set( $symbol_dollar = '$' )
+#set( $symbol_escape = '\' )
+package ${package}.util;
+
+import de.terrestris.shogun2.util.interceptor.WmsResponseInterceptorInterface;
+import de.terrestris.shogun2.util.model.Response;
+
+/**
+ * This class demonstrates how to implement the WmsResponseInterceptorInterface.
+ * 
+ * @author Daniel Koch
+ * @author terrestris GmbH & Co. KG
+ *
+ */
+public class WmsResponseInterceptor implements WmsResponseInterceptorInterface {
```

Comment:
Yeah, we could definitely do that. On top of that I would like to provide abstract classes that can be extended in the project application.